teams/tidb/membership.json

```diff
@@ -0,0 +1,73 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "HunDunDM",
```

Comment:
This list misses several maintainers and it may cause by the underneath of our bot logics.

I'd invite these maintainers to keep their seats by requesting, otherwise you will be regarded as emeritus (you can always request reinstatement). cc

* @ngaut 
* @c4pt0r 
* @qiuyesuifeng 
* @siddontang 
* @shenli 
* @zz-jason (current as committer from the membership file)
* @winkyao (current as committer from the membership file)
* @disksing 
* @coocood (current as committer from the membership file)

Reply:
how to request?

Reply:
We are under a re-bootstrap phase and I think you just request by comments here and the list will be reviewed at a whole by the TOC.

Reply:
> We are under a re-bootstrap phase and I think you just request by comments here and the list will be reviewed at a whole by the TOC.

Please add me. Thanks!

Reply:
I think we should propose a new maintainer list instead of making the leaders of previous sigs be maintainers, and the list needs to be approved by TiDB Community TOC.

Reply:
I think the right of the new maintainer org is not the same as sig leaders.

Reply:
Please keep me seat

Reply:
@winkyao I comment the role definition here https://github.com/pingcap/community/issues/519#issuecomment-900127665.

> A new sig (named as team for distinguished) has its reviewer, committer and maintainer.
> 
> * A reviewer is able to review pull request for quality and correctness of repositories associated with the team. His/Her LGTM is counted.
> * A committer is able to both review and commit pull request.
> * A maintainer, based on the rights of a committer, is able to promote or evict(emeritus member for reference) a reviewer, a committer or a maintainer by majority decision.
> 
> The maintainers is able to evict member by majority decision and thus we have an exit mechanism. It is suggested that a team defines its own bylaws to elaborate other team level decision making process.
> 
> Besides the teams, the whole TiDB community has its [TOC](https://github.com/pingcap/community/blob/master/toc/README.md) members for evolving the community. The TOC members are in charge of community level decision making process.

So a maintainer is a committer plus permission to make sig (team) level decision, such as membership modification and so on. The scope of permission is widen a lot from a SIG to a team, for from part of a repo to multiple repos. So it's deserved more discussion and if necessary, a meeting to discuss the list.

Reply:
@siddontang @disksing @winkyao https://github.com/pingcap/community/pull/522/commits/03d2a50c0a1d77d43780604b17e2c39a4dd43e70 respect all tidb founders.